requirements.txt

```diff
@@ -0,0 +1,94 @@
+alabaster==0.7.10
```

Comment:
Are these dependencies for iris-api itself or for iris-mobile? If they're for the latter I think they should go into iris-mobile's subfolder as setup.py currently handles dependencies for iris-api